frontend/appflowy_flutter/lib/workspace/presentation/home/menu/view/view_item.dart

```diff
@@ -148,6 +150,11 @@ class InnerViewItem extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final factor =
+        context.watch<AppearanceSettingsCubit>().state.fontIconsSizeFactor;
+    const tempOne = 1; // delete this later
+    final adjustableHeight = (factor + tempOne - 1) * 15;
```

Comment:
Is there a better way to allow the container's height to increase automatically instead of calculating it manually?

Reply:
I deleted the height from ViewItem, it will resize based on its font size. 
The minHeight is to make sure the height of the title is greater than the height of ViewAddButton and ViewMoreActionButton, so it won't expand when it is on hover. 

      child: Container(
        // minHeight to make sure the height of the title is greater than the height of ViewAddButton and ViewMoreActionButton
        constraints: const BoxConstraints(
          minHeight: 30,
        ),
        padding:
            EdgeInsets.fromLTRB(widget.level * widget.leftPadding, 2, 0, 2),
        child: Row(
          children: children,
        ),
      ),